Jakefile.js

```diff
@@ -307,7 +307,7 @@ function compileFile(outFile, sources, prereqs, prefixes, useBuiltCompiler, opts
         else {
             options += " --lib es5";
         }
-        options += " --noUnusedLocals --noUnusedParameters";
+        options += " --noUnusedLocals --noUnusedParameters --strictNullChecks";

```

Comment:
why not just `--strict`?

Reply:
and same in gulpfile

Reply:
and same in tsconfig.json

Reply:
`--strictPropertyInitialization` gives a ton of errors, as does `--strictFunctionTypes`.